fastparquet/api.py

```diff
@@ -122,36 +122,35 @@ def row_group_filename(self, rg):
         else:
             return self.fn
 
-
     def read_row_group_file(self, rg, columns, categories, index=None,
-                            assign=None):
+                            assign=None, mr_times=[]):
```

Comment:
This is common, it seems, in parquet implementations (like map-reduce and others derived from it) preceding the actual parquet standard timestamps. The real pity is that there is no hint in the schema about the int96 field's intent (could really simply be massive integers), which is why the user needs to specify.

I am not aware of other standards. That doesn't exclude the possibility. I can go with timestamp96.